sysadmin/Vagrantfile

```diff
@@ -0,0 +1,187 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+# All Vagrant configuration is done below. The "2" in Vagrant.configure
+# configures the configuration version (we support older styles for
+# backwards compatibility). Please don't change it unless you know what
+# you're doing.
+
+require 'yaml'
+settings = YAML.load_file 'vagrant_settings.yml'
+
+
+# Vagrantfile API/syntax version. Don't touch unless you know what you're doing!
+VAGRANTFILE_API_VERSION = "2"
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+
+  # required_plugins = %w( vagrant-vbguest vagrant-disksize )
```

Comment:
this commented out section is relevant to running Vagrant in conjunction with virtualbox, which has its own complications. I'll leave it commented out until one day there's a demand to get the virtualbox config working again.